config/routes.rb

```diff
@@ -3,5 +3,7 @@
   get "start/index"
   root "claims#new"
 
-  resources :claims, only: [:new, :create, :show, :update], param: :slug
+  constraints slug: /qts_year|claim_school/ do
```

Comment:
The `contraints` is a regex so this is saying 'qts_year or claim_school' only, just sanity checking my understanding

Reply:
yep, that's correct.